bin/functions/load_config.py

```diff
@@ -606,11 +610,10 @@ def probe_java_opts():
     cnt = 0
     map_java_opts_line = ""
     reduce_java_opts_line = ""
-    try:
-        with open(file_name) as f:
-            content = f.read()
-    except IOError:
-        return
+    lines = read_file_content(file_name)
```

Comment:
Can you still use try and catch the exception?  `read_file_content` is likely to throws an exception